exotica/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(exotica)
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
It's portable to other compilers. E.g. if you want to release EXOTica one day for Windows with MSVC.

Reply:
It goes along with bumping the ``cmake_minimum_required`` to ``3.1`` (which our version may require as well). I defer to Vlad on this one (not sure what the best choice here is).

Reply:
I made it compatible with 14.04 again a while back so it does build but is not tested under CI. I do not object to bumping the CMake minimum version to 3.1 - just wondering what the really preferred way of setting this would be right now.